app/controllers/results_controller.rb

```diff
@@ -356,4 +376,43 @@ def records
       params[:show] = nil
     end
   end
+
+  private def compute_rankings_by_region(rows, continent, country)
```

Comment:
Out of sheer laziness: Is this a 1:1 copy from the helper or did you modify anything?

Reply:
1:1 copy